src/main/java/org/fundacionjala/coding/cesar/FizzBuzz.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.cesar;
+
+import java.util.ArrayList;
+
+import java.util.stream.Collectors;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class FizzBuzz {
+    private static final int FIVE = 5;
+    private static final int THREE = 3;
+    /**
+     * @param range List from int.
+     * @return list String replace.
+     */
+    public ArrayList<String> doBuzzFizz(final ArrayList<Integer> range) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "range" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)